sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -385,8 +388,10 @@ def __repr__(self):
     return 'AfterProcessingTime(delay=%d)' % self.delay
 
   def on_element(self, element, window, context):
-    context.set_timer(
-        '', TimeDomain.REAL_TIME, context.get_current_time() + self.delay)
+    context.add_state(self.COUNT_TAG, 1)
+    if context.get_state(self.COUNT_TAG) == 1:
```

Comment:
Would this be the correct check?

```suggestion
    if context.get_state(self.COUNT_TAG) >= 1:
```

Reply:
I think that would make it the same as before, it would increase the timer for every element, we only want to increase if it's the first we see.

Reply:
I have modified the logic from a Counter to a State, we don't need to actually count the elements, just modify the timer with the first element in the pane